clonelists/Sony - PlayStation.json

```diff
@@ -2000,12 +2279,19 @@
 		"Onside Complete Soccer": [
 			["Complete Soccer Onside", 1]
 		],
+		"Option - Tuning Car Battle Spec R": [
+			["SuperLite 1500 Series - Option - Tuning Car Battle Spec R", 2]
+		],
 		"Oumagatoki (Premium Fan Disc)": [
 			["Oumagatoki", 2]
 		],
 		"Pac-Man World - 20th Anniversary": [
 			["Pac-Man World 20th Anniversary", 1]
 		],
+		"Pachi-Slot Aruze Oukoku 4 (PSone Books)": [
```

Comment:
`(PSone Books)` is already in `internal-config.json` as a promote tag. This means `Pachi-Slot Aruze Oukoku 4` covers both _Pachi-Slot Aruze Oukoku 4 (Japan)_  and _Pachi-Slot Aruze Oukoku 4 (Japan) (PSone Books)_. So this would look like this instead:

```
"Pachi-Slot Aruze Oukoku 4": [
			["Pachi-Slot Aruze Oukoku 4 (Deluxe Pack)", 2]
		],
```